configuration/eip/application.properties

```diff
@@ -1,78 +1,175 @@
-debezium.offsetFilename=offsetFilename.logs
-# Name of the file for debezium to store history
-debezium.historyFilename=historyFilename.logs
-
-# ================= Senaite configuration properties ===============================
 
+# *********************** Configuration of the Senaite Web App *******************************************
+#
 # URL to connect to the SENAITE REST API
 senaite.baseUrl=http://localhost:8080/senaite
+
 # SENAITE username
 senaite.username=admin
+
 # SENAITE password
 senaite.password=admin
 
-# ================= EIP specific configuration properties ===============================
+# Lab Order Type (Bahmni specific)
+bahmni.test.orderType.uuid=114614db-5ed4-11eb-b7a8-0242ac120007
+# ----------------------------------------------------------------------------------------------------------------------
 
-# Logging level configuration
-logging.level.root=INFO
-openmrs.eip.log.level=DEBUG
-logging.level.outbound-lims=${openmrs.eip.log.level}
 
-# The interval in milliseconds between polls of the retry queue by the retry route
-db-event.retry.interval=60000
-# The interval in milliseconds before the first poll of the retry queue by the retry route when the application starts.
-db-event.retry.initial.delay=60000
+# *********************** Configuration of the Management database *****************************************************
+#
+# Driver class of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.driverClassName=org.h2.Driver
 
-# The interval in milliseconds between polls to update ServiceRequest's Task's status by the inbound-lims route
-serviceRequest-task-status.update.interval=10000
-# The interval in milliseconds before the first poll to update ServiceRequest's Task's status by the inbound-lims route when the application starts.
-serviceRequest-task-status.update.initial.delay=10000
\ No newline at end of file
+# Dialect of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.dialect=org.hibernate.dialect.H2Dialect
+
+# Url of the management datasource
+spring.mngt-datasource.jdbcUrl=jdbc:h2:file:${eip.home}/db/custom/openmrs_eip_mgt
+
+# User name of the management datasource
+spring.mngt-datasource.username=name
+
+# Password of the management datasource
+spring.mngt-datasource.password=password
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Debezium Config ******************************************************************************
+#
+# For details explanations of this properties, please refer to
+# https://debezium.io/documentation/reference/connectors/mysql.html#mysql-connector-properties
+# Maps to the connect property named database.server.id, MUST match the server-id value in your my.cnf file when you
+# enabled mysql binlog, DO NOT change after setting it
+debezium.db.serverId=2
```

Comment:
You need to set this to your actual server-id as your my.cnf file

Reply:
Ohhhh, yes. The server-id is actually 2.